wordpress-mobile/WordPress-iOS
# [Stats Refresh] Add Insights > Publicize

Fixes #10563 

To test:

---
- On a site that has social connections, go to Stats > Insights.
  - Tip: en.blog does have at least a Twitter connection.
- Verify a row appears for each connection, and only the connections the site has. As designed:

<img width="376" alt="publicize" src="https://user-images.githubusercontent.com/1816888/49257378-5c88c280-f3ef-11e8-89e5-80b5e52aec82.png">

---
- On a site with no social connections, go to Stats > Insights.
- Verify the `No data yet` view appears.

<img width="465" alt="no_data" src="https://user-images.githubusercontent.com/1816888/49257342-3c590380-f3ef-11e8-8973-fd7fabafd57d.png">

---
NOTE: The Twitter image looks like this, which is the blavatar we're currently downloading:
<img width="451" alt="twitter" src="https://user-images.githubusercontent.com/1816888/49257577-f51f4280-f3ef-11e8-8070-2fa32e74ede2.png">
